lighthouse-core/scripts/compare-runs.js

```diff
@@ -205,14 +213,24 @@ function aggregateResults(name) {
   for (const lhrPath of glob.sync(`${outputDir}/**/lhr-*.json`)) {
     const lhrJson = fs.readFileSync(lhrPath, 'utf-8');
     /** @type {LH.Result} */
-    const lhr = JSON.parse(lhrJson);
+    const lhr = lhrJson && JSON.parse(lhrJson);
+    if (!lhr || !lhr.audits) {
+      console.warn(`could not parse ${lhrPath}`);
+      continue;
+    }
 
-    const metrics = lhr.audits.metrics ?
+    if (argv.urlFilter && !lhr.requestedUrl.includes(argv.urlFilter)) continue;
+
+    const metrics = lhr.audits.metrics && lhr.audits.metrics.details ?
```

Comment:
This is starting to get out of hand. `&&` and a ternary in 1 assignment 😟 

Am I too go-thonic to say 
```js
const metrics = {}
if (lhr.audits.metrics && lhr.audits.metrics.details) {
    metrics = /** @type {!LH.Audit.Details.Table} */ (lhr.audits.metrics.details).items[0];
}
```

Honestly curious if it's more readable one way or the other. I think once a ternary becomes complex at all it's time to separate it out.

Reply:
if only `lhr.audits.metrics?.details` ...

> Am I too go-thonic to say

i think it's gooey :P I would prefer it if the ternary were more complex, but as it is it's just a nullability check.